test/wpt/runner/runner.mjs

```diff
@@ -313,7 +313,7 @@ export class WPTRunner extends EventEmitter {
       `skipped: ${skipped}`
     )
 
-    process.exit(0)
+    process.exit(failed - expectedFailures ? 1 : 0)
```

Comment:
Just to ensure any otherwise set non-zero exitCode is not lost.

```suggestion
    process.exit(failed - expectedFailures ? 1 : process.exitCode)
```